docs/install_guides/pypi-openvino-dev.md

```diff
@@ -87,11 +85,11 @@ python -m pip install --upgrade pip
 
 To install and configure the components of the development package for working with specific frameworks, use the `pip install openvino-dev[extras]` command, where `extras` is a list of extras from the table below: 
 
-| DL Framework                                                                     | Extra                          |
+| DL Framework                                                                     | Extra                           |
 | :------------------------------------------------------------------------------- | :-------------------------------|
 |   [Caffe*](https://caffe.berkeleyvision.org/)                                    | caffe                           |
-|   [Caffe2*](https://caffe2.ai/)                                                 | caffe2                          |
-|   [Kaldi*](https://kaldi-asr.org/)                                               | kaldi                           |
+|   [Caffe2*](https://github.com/pytorch/pytorch)                                  | caffe2                          |
```

Comment:
It is unexpected to see Caffe2 link to PyTorch.

Reply:
Caffe2 is part of PyTorch now. See deprecation notice here: https://caffe2.ai/